fcrepo-kernel-modeshape/src/test/java/org/fcrepo/integration/kernel/modeshape/FedoraResourceImplIT.java

```diff
@@ -752,6 +755,8 @@ public void testReplacePropertiesHashURIs() throws RepositoryException {
         final Model updatedModel3 = object.getTriples(subjects, PROPERTIES).collect(toModel());
 
         updatedModel3.remove(subject, dcCreator, hashResource);
+        updatedModel3.removeAll(hashResource, (Property)null, (RDFNode)null);
```

Comment:
This line seemed peculiar, so I tested with it commented out.. with all tests passing. 
It appears that the update in this line can be removed/reverted.